src/components/FavoriteCommercesLIstItem.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { ListItem, Button } from 'react-native-elements';
+import { View } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import {
+  registerFavoriteCommerce,
+  deleteFavoritesCommerces,
+  readFavoriteCommerce
+} from '../actions/CommercesListActions';
+import { connect } from 'react-redux';
+
+class FavoriteCommercesListItem extends Component {
+  state = { favorite: false };
+
+  componentWillMount() {
+    console.log(this.props.favoritesCommerce);
+    this.setState({
+      favorite: this.props.favoritesCommerce.includes(this.props.commerce.id)
+    });
+  }
+
+  onFavoritePress = commerceId => {
+    if (this.props.favoritesCommerce.includes(commerceId)) {
```

Comment:
lo mismo que puse en el otro archivo, se podria usar directamente this.state.favorites y te ahorras de validar si esta en el array